config/settings.yml

```diff
@@ -1228,6 +1228,7 @@ covid_vaccine:
 
 virtual_agent:
   webchat_secret: fake_secret
+  webchat_pva2_secret: fake_secret
```

Comment:
Has this been added to the setting.yml definition in the values.yml over in the manifests repo for Vets API?

Reply:
We added the secret to secrets.yaml and values.yaml in the manifests repo, see [merged PR here](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2217/files)